apis/keda/v1alpha1/scaledobject_types.go

```diff
@@ -254,21 +260,49 @@ func (so *ScaledObject) GetHPAMaxReplicas() int32 {
 	return defaultHPAMaxReplicas
 }
 
+// GetDefaultHPAMaxReplicas returns defaultHPAMaxReplicas
+func (so *ScaledObject) GetDefaultHPAMaxReplicas() int32 {
+	return defaultHPAMaxReplicas
```

Comment:
same here, having this as a method on `ScaledObject` structure feels unnecessary